WordPress/Classes/Utility/ZendeskUtils.swift

```diff
@@ -81,26 +84,172 @@ import ZendeskSDK
 
         ZDKRequests.pushRequestList(with: navController, layoutGuide: ZDKLayoutRespectTop)
     }
+
+    static func createRequest() {
+
+        if !ZendeskUtils.identityCreated {
+            return
+        }
+
+        ZDKRequests.configure { (account, requestCreationConfig) in
+
+            guard let requestCreationConfig = requestCreationConfig else {
+                return
+            }
+
+            // Set Zendesk ticket form to use
+            ZDKConfig.instance().ticketFormId = TicketFieldIDs.form
+
+            // Set form field values
+            let zdUtilsInstance = ZendeskUtils.sharedInstance
+            var ticketFields = [ZDKCustomField]()
+            ticketFields.append(ZDKCustomField(fieldId: TicketFieldIDs.appVersion, andValue: ZendeskUtils.appVersion))
+            ticketFields.append(ZDKCustomField(fieldId: TicketFieldIDs.allBlogs, andValue: zdUtilsInstance.getBlogInformation()))
+            ticketFields.append(ZDKCustomField(fieldId: TicketFieldIDs.deviceFreeSpace, andValue: zdUtilsInstance.getDeviceFreeSpace()))
+            ticketFields.append(ZDKCustomField(fieldId: TicketFieldIDs.networkInformation, andValue: zdUtilsInstance.getNetworkInformation()))
+            ticketFields.append(ZDKCustomField(fieldId: TicketFieldIDs.logs, andValue: zdUtilsInstance.getLogFile()))
+            ZDKConfig.instance().customTicketFields = ticketFields
+
+            // Set tags
+            requestCreationConfig.tags = zdUtilsInstance.getTags()
+
+            // Set the ticket subject
+            requestCreationConfig.subject = Constants.ticketSubject
+        }
+    }
+
 }
 
 // MARK: - Private Extension
 
 private extension ZendeskUtils {
 
     func enableZendesk(_ enabled: Bool) {
-        let defaults = UserDefaults.standard
-        defaults.set(enabled, forKey: UserDefaultsKeys.zendeskEnabled)
-        defaults.synchronize()
+        ZendeskUtils.zendeskEnabled = enabled
         DDLogInfo("Zendesk Enabled: \(enabled)")
     }
 
-    struct UserDefaultsKeys {
-        static let zendeskEnabled = "wp_zendesk_enabled"
+    // MARK: - Data Helpers
+
+    func getDeviceFreeSpace() -> String {
+
+        var deviceFreeSpace = Constants.unknownValue
+
+        if let resourceValues = try? URL(fileURLWithPath: "/").resourceValues(forKeys: [.volumeAvailableCapacityKey]),
```

Comment:
Nitpicky, but if we wanted to save a line of code we could make this a guard. Up to you. :)